src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574666,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040002
+name: PIK3CA-related overgrowth spectrum
+def: "Any overgrowth syndrome resulting from pathogenic gain-of-function variants in the PIK3CA gene. The variants can be germline or somatic" [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:23592320, PMID:23946963, PMID:25557259]
+synonym: "overgrowth syndrome and/or cerebral malformations due to abnormalities in MTOR pathway genes" BROAD [https://clinicalgenome.org/affiliation/50020/, PMID:23592320, PMID:25557259]
+is_a: MONDO:0019716 {source="PMID:23592320", source="PMID:23946963", source="PMID:25557259", source="https://clinicalgenome.org/affiliation/50020/"} ! overgrowth syndrome
+property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T13:33:20Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```

Comment:
please add the term tracker item to link to the ticket:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item

Reply:
Comment addressed.